### ISSUE TITLE
Goto Job

### ISSUE THREAD

**Issue by bluecereal** [yeliknewo/kh-screeps#18]
sends a creep to a room

**Comment by bluecereal** [comment]
closing, goto job is not relevant and it does not seem likely to be so in the future.

**Comment by yeliknewo** [comment]
This will be relevant any time there is multiroom stuff.

**Comment by bluecereal** [comment]
Yes, but a goto job isn't necessary for transferring creeps betweens rooms.  I was thinking that inter-room creep travel could instead be facilitated by populating the targetPool of one room with targets from another.  
When a creep gets a target from another room, it moves to that room and uses the targetPool there, which could have targets in the creeps home room, thus making it possible to proxy mine a room.
